### PR TITLE
Python 3: port tests in html/infrastructure

### PR DESCRIPTION
**Pull request by ziransun**

### CODE REVIEW

File: html/infrastructure/urls/resolving-urls/query-encoding/resources/css-tmpl.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 def main(request, response):
-    encoding = request.GET['encoding']
-    tmpl = request.GET['tmpl']
-    sheet = tmpl % u'\\0000E5'
-    return [("Content-Type", "text/css; charset=%s" % encoding)], sheet.encode(encoding)
```
Comment (annevk) [inline html/infrastructure/urls/resolving-urls/query-encoding/resources/css-tmpl.py:5]: It seems you removed functionality here.

Comment (ziransun) [reply html/infrastructure/urls/resolving-urls/query-encoding/resources/css-tmpl.py:5]: Thanks very much for spotting this! I removed encode() in a couple of places some time ago while fiddling with the codes but forgot to put them back. 

Codes have now been updated.

**Review by annevk — APPROVED** [review]
Looks reasonable. @zcorpan might want to quickly skim this too.

**Comment by ziransun** [comment]
> Looks reasonable. @zcorpan might want to quickly skim this too.

@zcorpan any chance to have a quick look? Thanks!

**Review by zcorpan — APPROVED** [review]
Apologies for the delay. LGTM

**Pull request merged**